[Doc] Add Atlas 300I series tutorials (#1341)
vllm-project/vllm-ascend: docs/source/tutorials/single_node_300i.md

# Single Node (Atlas 300I series)

## Run vLLM on Altlas 300I series

Run docker container:

```{code-block} bash
   :substitutions:
# Update the vllm-ascend image

> export IMAGE=quay.io/ascend/vllm-ascend:main-310p

Reviewer: ```suggestion
export IMAGE=quay.io/ascend/vllm-ascend:|vllm_ascend_version|-310p
```

:::::{tab-set}
::::{tab-item} Qwen3-0.6B

```{code-block} bash
   :substitutions:
export VLLM_USE_V1=1
export MODEL="Qwen/Qwen3-0.6B"
python -m vllm.entrypoints.api_server \
    --model $MODEL \
    --tensor-parallel-size 1 \
    --max-num-batched-tokens 2048 \
    --gpu-memory-utilization 0.5 \
    --max-num-seqs 4 \
    --enforce-eager \
    --trust-remote-code \
    --max-model-len 1024 \
    --disable-custom-all-reduce \
    --dtype float16 \
    --port 8000 \
    --compilation-config '{"custom_ops":["+rms_norm", "+rotary_embedding"]}' 
```
::::

::::{tab-item} Qwen/Qwen2.5-7B-Instruct

```{code-block} bash
   :substitutions:
export VLLM_USE_V1=1
export MODEL="Qwen/Qwen2.5-7B-Instruct"
python -m vllm.entrypoints.api_server \
    --model $MODEL \
    --tensor-parallel-size 2 \
    --max-num-batched-tokens 2048 \
    --gpu-memory-utilization 0.5 \
    --max-num-seqs 4 \
    --enforce-eager \
    --trust-remote-code \
    --max-model-len 1024 \
    --disable-custom-all-reduce \
    --dtype float16 \
    --port 8000 \
    --compilation-config '{"custom_ops":["+rms_norm", "+rotary_embedding"]}' 
```
::::

> ::::{tab-item} pangu-72b-MoE

Reviewer: ```suggestion
::::{tab-item} Pangu-Pro-MoE-72B
```

```{code-block} bash
   :substitutions:
# Update the MODEL

> export MODEL=

Reviewer: ```suggestion
export MODEL=“/path/to/pangu-pro-moe-model”
```

            max_num_batched_tokens=2048,
            gpu_memory_utilization=0.5,
            max_num_seqs=4,

>             enforce_eager=True, # For ascend 310P eager mode, only single-operator model is supported.

Reviewer: ```suggestion
            enforce_eager=True, # For 300I series, only eager mode is supported.
```
